dart/lib/src/platform_checker.dart

```diff
@@ -1,7 +1,12 @@
+import 'package:platform/platform.dart';
```

Comment:
its a dart/google package but usually we are against dependencies.
https://develop.sentry.dev/sdk/philosophy/#dependencies-cost
lets do the Draft with it and see if we can get rid of it later, so we have a testable Draft
cc @bruno-garcia

Reply:
Yeah, it doesn't work on web so we need to get rid of it anyways

Reply:
We tend to avoid taking dependencies for multiple reasons as Manoel said, so if it's if it's enough to copy some snippets from that library to identify a platform, we can do that. Times I did this I linked to the original source code (with commit sha in the path instead of a branch name such as `main` so we can easily open the original file and check any changes/bug fixes that happened to it).